base/_mixins.scss

```diff
@@ -103,155 +146,351 @@
+/// @author jhogue
+///
+/// @param {list} Ratio [1 1]
+///
+/// @require {mixin} maintain-ratio
+///
+/// @example scss - Usage
+///  // @include proportional-container(16 9)
+///
+/// @example css - Output
+///  // position: relative; overflow: hidden; z-index: 1; height: 0; padding-bottom: 56.25%; width: 100%
+///
 @mixin proportional-container( $ratio: 1 1 ) {
-	overflow: hidden;
 	position: relative;
+	overflow: hidden;
 	z-index: 1;
 	@include maintain-ratio( $ratio );
 }
 
-// Use like you would background-cover, except on imgs inside a wrapper
-// Accepts '100% auto' (width height), 'auto 100%' (width height) or 'contain' as values
+/// image-cover() =
+/// Sets a container to "crop" an element to a fixed aspect ratio
+/// @author jhogue
+///
+/// @param {String} $center ['100% auto']
+///   Accepts '100% auto' (width height), 'auto 100%' (width height), or 'contain'
+///
+/// @require {library} Bourbon
+/// @require {mixin} transform
+///
+/// @example scss - Usage
+///  // @include image-cover(auto 100%)
+///
 @mixin image-cover( $center: '100% auto' ) {
 	$allowed: ('100% auto','auto 100%','contain');
-	@if not index($allowed, $center) {
-		@error "Keyword `#{$center}` for mixin 'image-cover' is not allowed. #{$allowed} is expected.";
+
+  @if not index($allowed, $center) {
+		@error "Keyword `#{$center}` for mixin 'image-cover' is not allowed. `#{$allowed}` is expected.";
 	}
 	position: absolute;
 	z-index: 2;
+
 	@if ($center == 'contain') {
+		// Keep the entire element inside the container
 		@include transform( translate(-50%,-50%) );
-		top: 50%; left: 50%;
-		height: auto;
-		max-height: 100%; // we already have a rule for max-width: 100%
+		top: 50%;
+		left: 50%;
 		width: auto;
+		height: auto;
+		max-height: 100%;
 	} @else if ($center == '100% auto') {
 		// Fill the width, let the height be what it needs to be
 		@include transform( translateY(-50%) );
-		top: 50%; left: 0;
-		height: auto;
+		top: 50%;
+		left: 0;
 		width: 100%;
+		height: auto;
 	} @else {
 		// Default: Fill the height, let the width be what it needs to be
 		@include transform( translateX(-50%) );
-		top: 0; left: 50%;
-		height: 100%;
-		max-width: 200%; // reset the max-width: 100% present on all img elements
+		top: 0;
+		left: 50%;
 		width: auto;
+		max-width: none;
+		height: 100%;
 	}
 }
 
-// Provide REM and PX fallback for any property
-// Gulp used to do this for us, but Brunch does not
-// @accepts: css property, numeric value
-// @requires: value must be a pixel value, stripped of its unit
-// @dependson: px(), a function of ours, and rem(), a function of Bourbon
-// @returns: two property values, one with px, one with rem units
-// @jhogue
+/// rem() =
+/// Provide REM and PX fallback for any property
+/// @author jhogue
+///
+/// @param {String} valid css property
+///   The property to calculate px and rem for
+///
+/// @param {Numeric} value
+///   The value, assumed ot be pixels
+///
+/// @require {function} unquote
+///
```

Comment:
in other mixins, the @require items are grouped together, here there are extra line breaks